api-tests/features/dl-auth-source-checks.feature

```diff
@@ -0,0 +1,183 @@
+@This
+Feature: Authoritative source checks with driving licence CRI
+
+  Scenario Outline: Journey through DCMAW with driving licence requires authoritative source check
+    Given I activate the 'drivingLicenceAuthCheck,p1Journeys' feature sets
+    When I start a new '<journey-type>' journey
+    Then I get a 'page-ipv-identity-document-start' page response
+    When I submit an 'appTriage' event
+    Then I get a 'dcmaw' CRI response
+    When I submit 'kenneth-driving-permit-valid' details to the CRI stub
+    Then I get a 'drivingLicence' CRI response
+    When I submit 'kenneth-driving-permit-valid' details with attributes to the CRI stub
+      | Attribute | Values          |
+      | context   | "check_details" |
+    Then I get a 'page-dcmaw-success' page response
+
+    Examples:
+      | journey-type       |
+      | low-confidence    |
+      | medium-confidence |
+
+  Scenario Outline: Journey with auth source check that attracts a CI leads to a mitigation journey
+    Given I activate the 'drivingLicenceAuthCheck,p1Journeys' feature sets
+    When I start a new '<journey-type>' journey
+    Then I get a 'page-ipv-identity-document-start' page response
+    When I submit an 'appTriage' event
+    Then I get a 'dcmaw' CRI response
+    When I submit 'kenneth-driving-permit-valid' details to the CRI stub
+    Then I get a 'drivingLicence' CRI response
+    When I submit 'kenneth-driving-permit-needs-alternate-doc' details with attributes to the CRI stub
+      | Attribute | Values          |
+      | context   | "check_details" |
+    Then I get a 'pyi-driving-licence-no-match-another-way' page response
```

Comment:
It would be nice if we could test that the reset-session lambda is run at this point, but we don't have a mechanism, and the lambda doesn't emit an audit event...

Reply:
Ah yeah, option 1 is the simplest. I'll add it in.